cms/djangoapps/contentstore/tests/test_contentstore.py

```diff
@@ -114,6 +114,40 @@ def test_xlint_fails(self):
         err_cnt = perform_xlint(TEST_DATA_DIR, ['toy'])
         self.assertGreater(err_cnt, 0)
 
+    def test_asset_with_invalid_displayname(self):
+        '''
+        This test verifies that when an asset has '/' in its displayname they are converted to underscores `_`
+        '''
+        content_store = contentstore()
+        course_assets = set([u'_invalid_displayname_subs-esLhHcdKGWvKs.srt'], )
```

Comment:
Agree, we can compare the displayname with expected displayname, then we will not need that set.